velox/type/Type.cpp

```diff
@@ -370,6 +370,18 @@ const TypePtr& RowType::findChild(folly::StringPiece name) const {
   VELOX_USER_FAIL(makeFieldNotFoundErrorMessage(name, names_));
 }
 
+bool RowType::isOrderable() const {
+  return std::all_of(children_.cbegin(), children_.cend(), [](const auto& c) {
+    return c->isOrderable();
+  });
+}
+
+bool RowType::isComparable() const {
+  return std::all_of(children_.cbegin(), children_.cend(), [](const auto& c) {
+    return c->isOrderable();
```

Comment:
should this be `c->isComparable()` instead?

Reply:
Fixed